data/brands/amenity/fast_food.json

```
@@ -1728,6 +1728,21 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Haldiram's",
+      "id": "haldirams-aba431",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "alt_name": "Haldiram",
+        "amenity": "fast_food",
+        "brand": "Goli Vada Pav",
```

Don't forget to change this tag too..
```suggestion
        "brand": "Haldiram's",
```

---

Hey @bhousel I have made the requested changes, please review.